### PR TITLE
Convert Less to Sass

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Convert Less files to Sass and bring in Sass files from https://github.com/cfpb/design-system/pull/1993

## How to test this PR

1. `yarn install && yarn build` and site should be unchanged.

### CODE REVIEW

File: cfgov/unprocessed/apps/form-explainer/css/form-explainer.scss [hunk 1]
```diff
@@ -103,38 +123,39 @@
...
 
-    each( @colors, {
-      @tint: @{value}-bg;
-      @light: @{value}-lt;
-      @med: @{value}-md;
+    @each $value in $colors {
+      $index: list.index($colors, $value);
+      $tint: #{$value}-bg;
+      $light: #{$value}-lt;
+      $med: #{$value}-md;
```
Comment (contolini) [inline cfgov/unprocessed/apps/form-explainer/css/form-explainer.scss:133]: Needs to read from colors map:

```suggestion
      $tint: map.get($colors-map, #{$value}-bg);
      $light: map.get($colors-map, #{$value}-lt);
      $med: map.get($colors-map, #{$value}-md);
```

Comment (anselmbradford) [reply cfgov/unprocessed/apps/form-explainer/css/form-explainer.scss:133]: Thank you!

**Review by contolini — APPROVED** [review]
Nice job! Re: our chat about `system-ui`, I've noticed that some of our apps ([TCCP](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/), [find a housing counselor](http://localhost:8000/find-a-housing-counselor/)) are prioritizing `system-ui` over `Avenir Next`, causing the font to be slightly different. Not sure why but something to be aware of.

image

**Comment by wpears** [comment]
>I've noticed that some of our apps ([TCCP](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/), [find a housing counselor](http://localhost:8000/find-a-housing-counselor/)) are prioritizing system-ui over Avenir Next

This seems like it's because Avenir is only defined in cfpb-typography, vs. `system-ui` being defined in cfpb-core and both of those apps import core but not typography.

I imagine that sass is choking on it in a way that a less `reference` import isn't

**Comment by anselmbradford** [comment]
Updated to v3 DS. Try it now. Thanks!

**Comment by anselmbradford** [comment]
Erp I'll need to fix the jest tests so they don't try to scoop up the scss files, but the site can still be checked locally.

**Review by contolini — APPROVED** [review]
Tests and looks great!

**Pull request merged**